<Type Name="AsynchronousOperationException" FullName="System.IdentityModel.Services.AsynchronousOperationException">
  <Metadata><Meta Name="ms.openlocfilehash" Value="c18d3535b9d86d612f1e84e703a2d5462494ba42" /><Meta Name="ms.sourcegitcommit" Value="756d085f27705e86604f1bba5f2086ee23761acf" /><Meta Name="ms.translationtype" Value="MT" /><Meta Name="ms.contentlocale" Value="ja-JP" /><Meta Name="ms.lasthandoff" Value="01/30/2019" /><Meta Name="ms.locfileid" Value="55351700" /></Metadata><TypeSignature Language="C#" Value="public class AsynchronousOperationException : Exception" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit AsynchronousOperationException extends System.Exception" />
  <TypeSignature Language="DocId" Value="T:System.IdentityModel.Services.AsynchronousOperationException" />
  <TypeSignature Language="VB.NET" Value="Public Class AsynchronousOperationException&#xA;Inherits Exception" />
  <TypeSignature Language="C++ CLI" Value="public ref class AsynchronousOperationException : Exception" />
  <TypeSignature Language="F#" Value="type AsynchronousOperationException = class&#xA;    inherit Exception" />
  <AssemblyInfo>
    <AssemblyName>System.IdentityModel.Services</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Exception</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute>
      <AttributeName>System.Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="0fede-101">非同期操作中にエラーが発生するとスローされる例外。</span><span class="sxs-lookup"><span data-stu-id="0fede-101">The exception that is thrown when an error occurs during an asynchronous operation.</span></span></summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="0fede-102"><see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="0fede-102">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="0fede-103"><see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="0fede-103">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="0fede-104">このコンストラクターは、新しいインスタンスの <xref:System.Exception.Message*> プロパティをシステム提供のエラー記述メッセージに初期化します。</span><span class="sxs-lookup"><span data-stu-id="0fede-104">This constructor initializes the <xref:System.Exception.Message*> property of the new instance to a system-supplied message that describes the error.</span></span> <span data-ttu-id="0fede-105">このメッセージには、現在のシステムのカルチャが考慮されます。</span><span class="sxs-lookup"><span data-stu-id="0fede-105">This message takes into account the current system culture.</span></span>  
  
 <span data-ttu-id="0fede-106">次の表では、新しいインスタンスの初期プロパティ値を示します。</span><span class="sxs-lookup"><span data-stu-id="0fede-106">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="0fede-107">プロパティ</span><span class="sxs-lookup"><span data-stu-id="0fede-107">Property</span></span>|<span data-ttu-id="0fede-108">[値]</span><span class="sxs-lookup"><span data-stu-id="0fede-108">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException>|<span data-ttu-id="0fede-109">`null`。</span><span class="sxs-lookup"><span data-stu-id="0fede-109">`null`.</span></span>|  
|<xref:System.Exception.Message>|<span data-ttu-id="0fede-110">ローカライズされたエラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="0fede-110">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.IdentityModel.Services.AsynchronousOperationException : Exception -&gt; System.IdentityModel.Services.AsynchronousOperationException" Usage="new System.IdentityModel.Services.AsynchronousOperationException innerException" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="innerException"><span data-ttu-id="0fede-111">現在の例外の原因である <see cref="T:System.Exception" />。</span><span class="sxs-lookup"><span data-stu-id="0fede-111">The <see cref="T:System.Exception" /> that is the cause of the current exception.</span></span> <span data-ttu-id="0fede-112"><paramref name="innerException" /> パラメーターが <see langword="null" /> でない場合は、内部例外を処理する <see langword="catch" /> ブロックで現在の例外が発生します。</span><span class="sxs-lookup"><span data-stu-id="0fede-112">If the <paramref name="innerException" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="0fede-113">システムが提供するエラー メッセージと、この例外の原因である内部例外への参照を使用して、<see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="0fede-113">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class with a system-supplied error message and a reference to the inner exception that is the cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="0fede-114">前の例外の直接の結果としてスローされる例外については、<xref:System.Exception.InnerException> プロパティに、前の例外への参照が格納されます。</span><span class="sxs-lookup"><span data-stu-id="0fede-114">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException> property.</span></span> <span data-ttu-id="0fede-115"><xref:System.Exception.InnerException>プロパティは、コンス トラクターに渡されるのと同じ値を返します。</span><span class="sxs-lookup"><span data-stu-id="0fede-115">The <xref:System.Exception.InnerException> property returns the same value that is passed into the constructor.</span></span>  
  
 <span data-ttu-id="0fede-116">次の表では、新しいインスタンスの初期プロパティ値を示します。</span><span class="sxs-lookup"><span data-stu-id="0fede-116">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="0fede-117">プロパティ</span><span class="sxs-lookup"><span data-stu-id="0fede-117">Property</span></span>|<span data-ttu-id="0fede-118">[値]</span><span class="sxs-lookup"><span data-stu-id="0fede-118">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException>|<span data-ttu-id="0fede-119">内部例外の参照。</span><span class="sxs-lookup"><span data-stu-id="0fede-119">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message>|<span data-ttu-id="0fede-120">エラーを説明するシステム提供のメッセージ。</span><span class="sxs-lookup"><span data-stu-id="0fede-120">A system-supplied message that describes the error.</span></span> <span data-ttu-id="0fede-121">このメッセージには、現在のシステムのカルチャが考慮されます。</span><span class="sxs-lookup"><span data-stu-id="0fede-121">This message takes into account the current system culture.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.IdentityModel.Services.AsynchronousOperationException : string -&gt; System.IdentityModel.Services.AsynchronousOperationException" Usage="new System.IdentityModel.Services.AsynchronousOperationException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="0fede-122">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="0fede-122">The error message that explains the reason for the exception.</span></span></param>
        <summary><span data-ttu-id="0fede-123">指定したエラー メッセージを使用して、<see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="0fede-123">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class with a specified error message.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="0fede-124">このコンス トラクターの初期化、<xref:System.Exception.Message>を使用して新しい例外のプロパティ、`message`パラメーター。</span><span class="sxs-lookup"><span data-stu-id="0fede-124">This constructor initializes the <xref:System.Exception.Message> property of the new exception using the `message` parameter.</span></span> <span data-ttu-id="0fede-125">`message` の内容は、ユーザーが理解できる内容にします。</span><span class="sxs-lookup"><span data-stu-id="0fede-125">The content of `message` is intended to be understood by humans.</span></span> <span data-ttu-id="0fede-126">このコンストラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認しておく必要があります。</span><span class="sxs-lookup"><span data-stu-id="0fede-126">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span>  
  
 <span data-ttu-id="0fede-127">次の表では、新しいインスタンスの初期プロパティ値を示します。</span><span class="sxs-lookup"><span data-stu-id="0fede-127">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="0fede-128">プロパティ</span><span class="sxs-lookup"><span data-stu-id="0fede-128">Property</span></span>|<span data-ttu-id="0fede-129">[値]</span><span class="sxs-lookup"><span data-stu-id="0fede-129">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException>|<span data-ttu-id="0fede-130">`null`。</span><span class="sxs-lookup"><span data-stu-id="0fede-130">`null`.</span></span>|  
|<xref:System.Exception.Message>|<span data-ttu-id="0fede-131">エラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="0fede-131">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected AsynchronousOperationException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; AsynchronousOperationException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberSignature Language="F#" Value="new System.IdentityModel.Services.AsynchronousOperationException : System.Runtime.Serialization.SerializationInfo * System.Runtime.Serialization.StreamingContext -&gt; System.IdentityModel.Services.AsynchronousOperationException" Usage="new System.IdentityModel.Services.AsynchronousOperationException (info, context)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info"><span data-ttu-id="0fede-132">シリアル化されたオブジェクト データを保持する <see cref="T:System.Runtime.Serialization.SerializationInfo" /> オブジェクト。</span><span class="sxs-lookup"><span data-stu-id="0fede-132">A <see cref="T:System.Runtime.Serialization.SerializationInfo" /> object that holds the serialized object data.</span></span></param>
        <param name="context"><span data-ttu-id="0fede-133">転送元または転送先に関するコンテキスト情報を含んでいる <see cref="T:System.Runtime.Serialization.StreamingContext" /> オブジェクト。</span><span class="sxs-lookup"><span data-stu-id="0fede-133">A <see cref="T:System.Runtime.Serialization.StreamingContext" /> object that contains the contextual information about the source or destination.</span></span></param>
        <summary><span data-ttu-id="0fede-134">シリアル化したデータを使用して、<see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="0fede-134">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class with serialized data.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="0fede-135">このコンストラクターは、逆シリアル化中に、ストリームで送信された例外オブジェクトを再構築するために呼び出されます。</span><span class="sxs-lookup"><span data-stu-id="0fede-135">This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.IdentityModel.Services.AsynchronousOperationException : string * Exception -&gt; System.IdentityModel.Services.AsynchronousOperationException" Usage="new System.IdentityModel.Services.AsynchronousOperationException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="0fede-136">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="0fede-136">The error message that explains the reason for the exception.</span></span></param>
        <param name="innerException"><span data-ttu-id="0fede-137">現在の例外の原因である <see cref="T:System.Exception" />。</span><span class="sxs-lookup"><span data-stu-id="0fede-137">The <see cref="T:System.Exception" /> that is the cause of the current exception.</span></span> <span data-ttu-id="0fede-138"><paramref name="innerException" /> パラメーターが <see langword="null" /> でない場合は、内部例外を処理する <see langword="catch" /> ブロックで現在の例外が発生します。</span><span class="sxs-lookup"><span data-stu-id="0fede-138">If the <paramref name="innerException" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="0fede-139">指定したエラー メッセージおよびこの例外の原因となった内部例外への参照を使用して、<see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="0fede-139">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="0fede-140">前の例外の直接の結果としてスローされる例外については、<xref:System.Exception.InnerException> プロパティに、前の例外への参照が格納されます。</span><span class="sxs-lookup"><span data-stu-id="0fede-140">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException> property.</span></span> <span data-ttu-id="0fede-141"><xref:System.Exception.InnerException>プロパティは、コンス トラクターに渡されるのと同じ値を返します。</span><span class="sxs-lookup"><span data-stu-id="0fede-141">The <xref:System.Exception.InnerException> property returns the same value that is passed into the constructor.</span></span>  
  
 <span data-ttu-id="0fede-142">次の表では、新しいインスタンスの初期プロパティ値を示します。</span><span class="sxs-lookup"><span data-stu-id="0fede-142">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="0fede-143">プロパティ</span><span class="sxs-lookup"><span data-stu-id="0fede-143">Property</span></span>|<span data-ttu-id="0fede-144">[値]</span><span class="sxs-lookup"><span data-stu-id="0fede-144">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException>|<span data-ttu-id="0fede-145">内部例外の参照。</span><span class="sxs-lookup"><span data-stu-id="0fede-145">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message>|<span data-ttu-id="0fede-146">エラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="0fede-146">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>